diff --git a/etc/conf.d/connman b/etc/conf.d/connman
index c5d9939..c19e4f5 100644
--- a/etc/conf.d/connman
+++ b/etc/conf.d/connman
@@ -4,7 +4,7 @@
 #cfgfile=/etc/connman/main.conf

 # Additional arguments to pass to connmand.
-#command_args=
+command_args="-r"

 # Number of milliseconds to wait after starting and check that daemon is
 # still running.
diff --git a/etc/init.d/connman b/etc/init.d/connman
index 8fb0b55..49e7118 100644
--- a/etc/init.d/connman
+++ b/etc/init.d/connman
@@ -23,5 +23,10 @@ supervise_daemon_args=" $supervise_daemon_args"
 depend() {
 	need dbus
 	provide net
+	after dbus
+}
+
+start_pre() {
+	sleep 2
 }
 
