diff --git a/etc/rc.conf b/etc/rc.conf
index 03939e2..d91dc28 100644
--- a/etc/rc.conf
+++ b/etc/rc.conf
@@ -200,7 +200,7 @@ rc_tty_number=12
 # cgroups version 1 on /sys/fs/cgroup.
 # "legacy" mounts cgroups version 1 on /sys/fs/cgroup
 # "unified" mounts cgroups version 2 on /sys/fs/cgroup
-#rc_cgroup_mode="unified"
+rc_cgroup_mode="hybrid"

 # This is a list of controllers which should be enabled for cgroups version 2
 # when hybrid mode is being used.
